### PR TITLE
added .net core console sample app

### PR DESCRIPTION
**Pull request by Sudhaushu**
added .net core console sample app
made changes to .nuspec and .target for nuget to be compatible with .net core

### CODE REVIEW

File: samples/SentimentAnalyzerCustomSkill/cs/FaceSentimentAnalysisConsole/FaceSentimentAnalysisConsole.csproj [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <ApplicationManifest>app.manifest</ApplicationManifest>
+    <Platforms>AnyCPU;x64</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Contoso.FaceSentimentAnalyzer_CPP" Version="0.0.0.5" />
+    <PackageReference Include="Microsoft.AI.Skills.SkillInterfacePreview" Version="0.5.2.12" />
```
Comment (LPBourret) [inline samples/SentimentAnalyzerCustomSkill/cs/FaceSentimentAnalysisConsole/FaceSentimentAnalysisConsole.csproj]: >2 [](start = 90, length = 1)

13

File: samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_cpp/FaceSentimentAnalysisConsole_cpp.vcxproj [hunk 2]
```diff
@@ -187,12 +187,12 @@
...
   </ImportGroup>
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.12\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.12\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets'))" />
+    <Error Condition="!Exists('..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.13\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.13\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets'))" />
   </Target>
```
Comment (LPBourret) [inline samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_cpp/FaceSentimentAnalysisConsole_cpp.vcxproj]: this solution and the other CPP projects don't compile for me.. let's sync tomorrow.

Comment (Sudhaushu) [reply samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_cpp/FaceSentimentAnalysisConsole_cpp.vcxproj]: just pushed a fix.

File: samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalyzer/packages.config [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AI.Skills.SkillInterfacePreview" version="0.5.2.12" targetFramework="native" />
+  <package id="Microsoft.AI.Skills.SkillInterfacePreview" version="0.5.2.13" targetFramework="native" />
```
Comment (dhung-msft) [inline samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalyzer/packages.config]: Should this be 0.5.2.15?

File: samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_cpp/FaceSentimentAnalysisConsole_cpp.vcxproj (hunk 2 above)
Comment (dhung-msft) [reply samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_cpp/FaceSentimentAnalysisConsole_cpp.vcxproj]: The 0.5.2.13 should be 0.5.2.15.

Seems the SkillInterfacePreview was improperly rev'd up across the projects

File: samples/SentimentAnalyzerCustomSkill/cpp/scripts/AppWinrtCPP_PostBuild.ps1 [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+{
+    $redistPathDebug = $VCInstallDir + "Redist\MSVC\14.16.27012\onecore\debug_nonredist\$Arch\Microsoft.VC141.DebugCRT\"
+    foreach($file in $redistfiles)
+    {
+        $fullpath = $redistPathDebug + $file +"d.dll"
+        copy -Force $fullpath $TargetDir
+    }
+}
+
+#*_app.dll symlinks for the skills which are built with the APPCONTAINER flag, assuming skills are all in Release mode only
```
Comment (dhung-msft) [inline samples/SentimentAnalyzerCustomSkill/cpp/scripts/AppWinrtCPP_PostBuild.ps1]: Please put this as a >NOTE in a README somewhere

Comment (dhung-msft) [reply samples/SentimentAnalyzerCustomSkill/cpp/scripts/AppWinrtCPP_PostBuild.ps1]: Please also add a note about needing VS installed for ucrt debug DLLs to run

File: samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_winrtcpp/packages.config [hunk 5]
```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.AI.Skills.SkillInterfacePreview" version="0.5.2.12" targetFramework="native" />
+  <package id="Contoso.FaceSentimentAnalyzer_CPP" version="0.0.0.6" targetFramework="native" />
+  <package id="Microsoft.AI.Skills.SkillInterfacePreview" version="0.5.2.15" targetFramework="native" />
```
[the remark this answers is not in the document]
Comment (Sudhaushu) [reply samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysisConsole_winrtcpp/packages.config]: doesn't this reflect the ingested nuget? This file is also a dependency for the scripts

File: samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysis_cpp/FaceSentimentAnalysis_cpp.vcxproj [hunk 6]
```diff
@@ -187,12 +187,12 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
-    <Import Project="..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.12\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets" Condition="Exists('..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.12\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets')" />
+    <Import Project="..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.13\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets" Condition="Exists('..\packages\Microsoft.AI.Skills.SkillInterfacePreview.0.5.2.13\build\native\Microsoft.AI.Skills.SkillInterfacePreview.targets')" />
```
[the remark this answers is not in the document]
Comment (Sudhaushu) [reply samples/SentimentAnalyzerCustomSkill/cpp/FaceSentimentAnalysis_cpp/FaceSentimentAnalysis_cpp.vcxproj]: i thought i already changed this...let me verify

File: samples/SentimentAnalyzerCustomSkill/build/SentimentAnalyzer_CPP.nuspec [hunk 7]
```diff
@@ -3,51 +3,51 @@
     <metadata>
         <!-- Required elements-->
         <id>Contoso.FaceSentimentAnalyzer_CPP</id>
-        <version>0.0.0.4</version>
+        <version>0.0.0.5</version>
```
[the remark this answers is not in the document]
Comment (Sudhaushu) [reply samples/SentimentAnalyzerCustomSkill/build/SentimentAnalyzer_CPP.nuspec]: are you looking at older version of this file?

**Pull request merged**